### PR TITLE
feat: Sw 20 create carrers component

### PR DESCRIPTION
**Pull request by Jhow-Rambo**
Connected to

**File Created/updated:** #25 

**Notes:**

Create a careers component.

**Checklist:**

- [x] I added github label for semantic versioning
- [x] I double checked it looks like the designs
- [x] I completed any required mobile breakpoint styling
- [x] I completed any required hover state styling
- [x] I included a working spec file
- [x] I added notes above about how long it took to build this component
- [x] UX has reviewed this PR
- [x] I assigned this PR to someone to review

### CODE REVIEW

**Comment by matheusgomes062** [comment]
The mobile version of Carrers title needs to have a smaller icon: 
ideal: 
image

yours: 
image

File: components/Careers.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      data-cy="careers"
+    >
+      <SectionTitle
+        sectionTitle={props.sectionTitle}
+        description={props.description}
+        mode={props.mode}
+      />
+
+      <div className="flex flex-wrap w-full mt-8 mb-10">
+        {MockJobOpportunities.slice(0, 3).map((vacancy) => (
```
Comment (matheusgomes062) [inline components/Careers.tsx:25]: These shouldn't be here, please place the mock array on the storybook file, not in the component.

File: components/SolutionsSection.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const SolutionsSection: FunctionComponent = ({ ...props }) => {
       >
         <div className="mt-14">
           <SectionTitle
-            title="Soluções"
+            sectionTitle="Soluções"
```
Comment (matheusgomes062) [inline components/SolutionsSection.tsx:21]: This is something that should go in the storybook, not on the component itself (I think)

**Review by matheusgomes062 — CHANGES_REQUESTED** [review]
requests made. Excellent job overall! Very proud of you young padawan!

File: components/Careers.tsx (hunk 1 above)
Comment (Jhow-Rambo) [reply components/Careers.tsx:25]: Done. It was necessary to change the MoreOpportunities component too, it also used this Mock created and as the Mock was deleted the component broke.

**Pull request merged**